internal/pkg/cli/storage_init.go

```diff
@@ -476,7 +476,7 @@ func (o *initStorageOpts) createAddon() error {
 	)
 	log.Infoln(color.Help(`The Cloudformation template is a nested stack which fully describes your resource,
```

Comment:
I wonder if we should remove this help message?

We already recommend the user that they should run `copilot deploy` as a recommended action to deploy the addon as well as outputing the env variable name. I feel like this blurb is distracting them from that essential information.

Reply:
Yeah, we should have already walked customers through those kinds of concepts-- they can check out our docs for more description of the addons directory.